lib/image-inspector-client.rb

```diff
@@ -51,12 +51,7 @@ def fetch_metadata
     def exception_handler
       yield
     rescue RestClient::Exception => e
-      begin
-        json_error_msg = JSON.parse(e.response || '') || {}
-      rescue JSON::ParserError
-        json_error_msg = {}
-      end
-      err_message = json_error_msg['message'] || e.to_s
+      err_message = e.response || e.message || ''
```

Comment:
I agree, This is also a possible solution.
We can leave the deleted part and have 
```
err_message = json_error_msg['message'] || e.response || e.message || ''
```